libs/expo/shared/ui-components/src/lib/PillContainer/PillContainer.tsx

```diff
@@ -0,0 +1,57 @@
+import { ChevronLeftIcon } from '@monorepo/expo/shared/icons';
+import { Spacings } from '@monorepo/expo/shared/static';
+import { useState } from 'react';
+import { Pressable, StyleSheet, View } from 'react-native';
+import Pill from '../Pill';
+import TextBold from '../TextBold';
+
+export function PillContainer({
+  data,
+  type,
+  maxVisible,
+}: {
+  data: string[];
+  type?: 'primary' | 'success';
+  maxVisible: number;
```

Comment:
nice! 
aprvd. if wanna make 1 more change you can also maxVisible default to a 5, if that'd be our standard. but can do later